packages/flutter/lib/src/rendering/object.dart

```diff
@@ -648,6 +649,20 @@ class SemanticsHandle {
   }
 }
 
+@immutable
+class _ElevationData {
+  const _ElevationData(this.elevation, this.area, this.path, this.object)
```

Comment:
Even though it's private, probably not a bad idea to leave a quick doc comment, e.g. Stores that [object] has is drawn at [elevation] above its parents and <something with area and path>.

Reply:
Done